pkg/apis/sessions/session_state.go

```diff
@@ -22,6 +22,7 @@ type SessionState struct {
 	AccessToken  string `msgpack:"at,omitempty"`
 	IDToken      string `msgpack:"it,omitempty"`
 	RefreshToken string `msgpack:"rt,omitempty"`
+	ProviderID   string `msgpack:"pi,omitempty"`
```

Comment:
What would happen if an admin were to change the provider ID, I assume all sessions would get kicked out and you'd need to sign in again?

Reply:
Will test this and let you know